src/symmetricDifferenceWith.js

```diff
@@ -0,0 +1,29 @@
+var _curry3 = require('./internal/_curry3');
+var differenceWith = require('./differenceWith');
+var unionWith = require('./unionWith');
+
+
+/**
+ * Finds the set (i.e. no duplicates) of all elements contained in the first or
+ * second list, but not both.  Duplication is determined according to the value
```

Comment:
Word. There's a lot of this in the existing docstrings. I can fix those as well if preferred.